diagnostic_feedback/public/js/src/studio/quiz.js

```diff
@@ -343,7 +343,9 @@ function Quiz(runtime, element, initData) {
               studioCommon.hideAddGrpPanel(el);
               studioCommon.updateAllGroups(response.grp_name);
               studioCommon.updateAllResultGroupDropwdowns();
-              $(el).parent().parent().next().find('select').first().val(response.grp_name).change();
+
+              $(el).parents('.new-grp-container').next().filter('.existing-grps-list').find('select').first()
```

Comment:
@jareerahsan I don't think you need that `filter` call here: `.existing-grps-list` is a sibling of `.new-grp-container`, so you can just do

```js
$(el).parents('.new-grp-container').next('.existing-grps-list').find('select').first() // ...
```